regexec.c

```diff
@@ -3833,7 +3833,7 @@ Perl_regexec_flags(pTHX_ REGEXP * const rx, char *stringarg, char *strend,
 
     /* Check validity of program. */
     if (UCHARAT(progi->program) != REG_MAGIC) {
-        Perl_croak(aTHX_ "corrupted regexp program");
+        croak("corrupted regexp program");
```

Comment:
Panic.

Reply:
That sounds like a good suggestion for another branch. In this branch I'm aiming not to have any behavioral changes.